packages/lisk-cryptography/benchmark/nacl.js

```diff
@@ -0,0 +1,143 @@
+/*
+* Copyright © 2018 Lisk Foundation
+*
+* See the LICENSE file at the top-level directory of this distribution
+* for licensing information.
+*
+* Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+* no part of this software, including this file, may be copied, modified,
+* propagated, or distributed except according to the terms contained in the
+* LICENSE file.
+*
+* Removal or modification of this copyright notice is prohibited.
+*
+*/
+/* eslint-disable no-console */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Benchmark from 'benchmark';
```

Comment:
do we need this as part of the `cryptography` library? or just add it as part of tests in `cryptography` that way it need not be included in the package.

Reply:
since this is not exported and not build, it will not be included in the package.